src/extension/commandPaletteHandler.ts

```diff
@@ -88,7 +88,7 @@ export class CommandPaletteHandler {
     private executeReactNativeRunCommand(command: string, args?: string[]): Q.Promise<void> {
         // Start the packager before executing the React-Native command
         let outputChannel = vscode.window.createOutputChannel("React-Native");
-        Log.appendStringToOutputChannel("Attempting to start the React Native packager", outputChannel);
+        Log.logToOutputChannel(outputChannel, "Attempting to start the React Native packager");
```

Comment:
We are initializing the global logger with the output channel in this process, so we can just call Log.logMessage here, and remove logToOutputChannel given that this is the only caller.